java/daikon/DynComp.java

```diff
@@ -59,7 +67,10 @@ public class DynComp {
   // Null if shouldn't do output
   public static @MonotonicNonNull File trace_file = null;
 
-  /** Controls size of the stack displayed in tracing the interactions that occurred. */
+  /**
+   * Controls size of the stack displayed in tracing the interactions that occurred. Used in {@link
+   * TagEntry}.
+   */
   @Option("Depth of call hierarchy for line tracing")
   public static int trace_line_depth = 1;
 
```

Comment:
Just below, does setting `abridged_vars` to true admit the possibility of a name clash and a crash?

Reply:
I don't think so.  abridged_vars is only used to print simplfied output; internal variable is unchanged.